main.py

```diff
@@ -129,6 +210,23 @@ def checkEntailment(relation: Relation, query: Query, xml_io: XMLIO | None):
     return answer
 
 
+def simpleEntailment(relation: Relation, query: Query, xml_io: XMLIO | None):
+    schema, table = setUpSimpleTable(relation, query)
+    stepNum = 1
+    while True:
```

Comment:
Maybe we should change to this? Because right now the last intermediate step is the same as the result.
```suggestion
    while not satisfySimpleRequirement():
```